pokemongo_bot/__init__.py

```diff
@@ -30,11 +32,41 @@ def __init__(self, config):
     def start(self):
         self._setup_logging()
         self._setup_api()
-        self.stepper = Stepper(self)
+        self.step_walker = StepWalker(self)
+        self.navigator = SpiralNavigator(self)
         random.seed()
 
     def take_step(self):
-        self.stepper.take_step()
+        location = self.navigator.take_step()
+        cells = self.find_close_cells(*location)
+        self.work_on_cell(cells[0], location, False)
```

Comment:
What's the plan to have this look at more than just the first cell? Could we change `find_close_cells` to `get_current_metacell` or something?